local-db-sync.md

```diff
@@ -0,0 +1,128 @@
+# Orderbook CLI
+
+## Index Orderbook Command
+
+Indexes the given orderbook contract.
+
+### Parameters
```

Comment:
aren't these the same parameters that would be required locally also? and why not index all orderbooks in the settings?